README.md

```diff
@@ -39,7 +40,7 @@ Add this dependency to your project's POM:
 <dependency>
   <groupId>com.adyen</groupId>
   <artifactId>adyen-java-api-library</artifactId>
-  <version>9.0.0</version>
+  <version>9.0.1</version>
```

Comment:
we made some breaking changes here: https://github.com/Adyen/adyen-java-api-library/pull/438
so we need to upgrade to a major version, can you please bump it to 10.0.0?

Reply:
done. pushed.